public/src/components/ToastManager.jsx

```diff
@@ -0,0 +1,198 @@
+import { useState, useRef } from 'react';
+import * as React from 'react';
+import { PropTypes } from 'prop-types';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faCheckCircle, faExclamationTriangle, faTimes,
+} from '@fortawesome/free-solid-svg-icons';
+
+const getCSSForFlavor = (flavor) => {
```

Comment:
You should really build out a theme instead of having a bunch of colors thrown around

Reply:
Fixed!